kafka/json_schemas/message.json

```diff
@@ -11,13 +11,22 @@
   },
   "properties": {
     "key": {
-      "$ref": "https://raw.githubusercontent.com/asyncapi/asyncapi-node/v2.7.7/schemas/2.0.0.json#/definitions/schema",
+      "oneOf": [
+        {
+          "description": "Constant value of key.",
+          "type": "string"
+        },
+        {
+          "description": "Schema that describes the allowed values of key.",
+          "$ref": "https://raw.githubusercontent.com/asyncapi/asyncapi-node/v2.7.7/schemas/2.0.0.json#/definitions/schema"
```

Comment:
Should be updated to `https://raw.githubusercontent.com/asyncapi/asyncapi-node/v2.14.0/schemas/2.4.0.json#/definitions/schema`